### PR TITLE
BUG: characterize=False

### PR DESCRIPTION
**Pull request by nkeim**
When `locate()` called `refine()`, it assumed a certain order of keyword args. At some point `refine()` was changed. The result is that the value of `characterize` was not getting passed.

This PR adds a failing test, then fixes the bug.

This is arguably a fix that should be included in v0.3.

### CODE REVIEW

**Comment by danielballan** [comment]
Agreed. In a post-py2 world, many of these should become keyword-only args.

**Comment by nkeim** [comment]
Tests are failing because x, y, mass appear to depend on `characterize`. Is that expected?

**Comment by danielballan** [comment]
Hmm. I see the error. I'm not sure how that should depend on `characterize`.

File: trackpy/tests/test_feature.py [hunk 1]
```diff
@@ -428,6 +428,15 @@ def test_subpx_precision(self):
...
         assert_allclose(actual, expected, atol=PRECISION)
 
+    def test_nocharacterize(self):
+        pos = gen_nonoverlapping_locations((200, 300), 9, 5)
+        image = draw_spots((200, 300), pos, 5)
+        f_c = tp.locate(image, 5, engine=self.engine, characterize=True)
+        f_nc = tp.locate(image, 5, engine=self.engine, characterize=False)
+
+        assert len(f_nc.columns) == 3
+        assert_allclose(f_c.values[:3], f_nc.values[:3])
```
Comment (caspervdw) [inline trackpy/tests/test_feature.py]: You need to slice the feature ndarrays on the second axis, now there is a shape mismatch. I would say, don't slice f_nc and do [:, :3] for f_c.

**Comment by caspervdw** [comment]
Good catch! The test has a bug, see line note. x, y, mass should certainly not depend on characterize.

**Comment by nkeim** [comment]
@caspervdw Thanks! That seems to fix it.

**Pull request merged**